app/Services/Geocoder/GeocoderInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\Geocoder;
+
+interface GeocoderInterface {
+    public function geocode(string $address): ?array;
```

Comment:
Try to make (atleast) the return value an ValueObject like `Position` This way you would always know what is returned and what it holds.

Reply:
where would you define the class for this object?

Reply:
I think I'll make a `GeolocationHelper` under `Helpers`. The class can then also contain a method to serialize as a mysql point.

Reply:
Normally you would have a folder "ValueObjects" to contain these objects. But since we dont have that (yet) I can  live with `Data` to